app/controlplane/api/jsonfilter/v1/jsonfilter.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package jsonfilter.v1;
+
+import "buf/validate/validate.proto";
+
+option go_package = "github.com/chainloop-dev/chainloop/app/controlplane/api/jsonfilter/v1";
+
+// JSONFilter represents a filter for JSON fields.
+message JSONFilter {
+  string field_path = 1 [(buf.validate.field).string.min_len = 1];
+  JSONOperator operator = 2 [(buf.validate.field).enum = {defined_only: true}];
+  optional string value = 43;
+}
+
+// JSONOperator represents supported JSON filter operators.
+enum JSONOperator {
+  JSON_OPERATOR_UNSPECIFIED = 0;
```

Comment:
Looking at the payload, I see these operators too verbose. I'd just set them to "EQ", "NEQ".

Reply:
I know but I think the protobuf linter will complain if the enum values are not prefixed by the name of the enum. I believe it's also recommended [here](https://protobuf.dev/programming-guides/style/), section Enums:
```
Name collisions are a high risk when enums are defined at the top level of a file (not nested inside a message definition); in that case the siblings include enums defined in other files that set the same package, where protoc may not be able to detect the collision has occurred at code generation time.

To avoid these risks, it is strongly recommended to do one of:

    Prefix every value with the enum name (converted to UPPER_SNAKE_CASE)
    Nest the enum inside a containing message
```

Reply:
Thanks for the clarification. Then I guess the only solution is to nest `JSONOperator` inside another message, but it may not worth it or apply.